homework-submission/homework-week-4/src/components/week3-component/people.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import './people.css';
+import logo from './logo.svg';
+
+// Main Component
+class People extends Component {
+  constructor() {
+    super();
+    this.state = {
+      people: []
+    };
+  }
+
+  componentDidMount() {
+    fetch('https://uinames.com/api/?amount=10')
+      .then(res => res.json())
+      .then(people =>
+        this.setState(
+          {
+            isLoaded: true,
+            people
+          },
+          () => console.log('People fetched..', people)
+        )
+      );
+  }
+
+  render() {
+    if (!this.state.isLoaded) {
+      return <div>Loading...</div>;
+    }
+    return (
+      <React.Fragment>
+        <header className="App-header">
+          <img src={logo} className="App-logo" alt="logo" />
+          <h1 className="App-title">Shuffle People !</h1>
+        </header>
+        <ul>
+          {this.state.people.map((person, i) => (
+            <div>
```

Comment:
The `key` should be placed on the first child of the `map` function, in this case the `<div>`. Settings the keys on the children of the div is giving errors in the dev tools. Here is some reading material on keys; https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
Keys have been deleted from child elements of div, then added to the div.